libs/controllers/src/auth/index.ts

```diff
@@ -56,8 +58,9 @@ export class AuthController {
 
   @UseGuards(JwtAuthGuard)
   @Get('me')
-  getUsers() {
-    return this.appService.profile;
+  async getUsers(@Request() req: TRequestAuth) {
+    const { nik, email } = req.user;
+    return await this.appService.profile(nik, email);
```

Comment:
const profile = ({
nik,
email
}:  TRequestAuth)


type TRequestAuth = {
 email: string
 nik: string
}

const { nik.email} = req.user
this.appService.profile(req?.user)